cfme/services/myservice/ssui.py

```diff
@@ -140,14 +140,10 @@ def is_displayed(self):
 
 @MiqImplementationContext.external_for(MyService.update, ViaSSUI)
 def update(self, updates):
-    view = navigate_to(self, 'Edit')
+    view = navigate_to(self, 'Edit', wait_for_view="True")
```

Comment:
**REQUIRED**: use ``wait_for_view=True`` (boolean). Applies to other places where this is used as well.